WeatherToday/Sources/Presentation/Modules/WeatherViewController.swift

```diff
@@ -1,13 +1,13 @@
-import UIKit
 import RxSwift
+import UIKit
 
 final class WeatherViewController: UIViewController {
 
-    private lazy var cityTextField: UITextField = UITextField()
-    private lazy var temperatureLabel: UILabel = UILabel()
-    private lazy var humidityLabel: UILabel = UILabel()
-    private var landscapeTopConstraint: NSLayoutConstraint = NSLayoutConstraint()
-    private var portraitTopConstraint: NSLayoutConstraint = NSLayoutConstraint()
+    private lazy var cityTextField: UITextField = .init()
```

Comment:
I think this is easier to read, also pass lint.
```suggestion
    private lazy var cityTextField = UITextField()
```

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/23/commits/c01b2420a65d92ab54c11dac43f4a1a23d2adece)